api/build.gradle

```diff
@@ -334,3 +334,40 @@ version = '0.1.0'          // Version in generated output
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
+
+
+class IncrementalHotSwapTask extends DefaultTask {
```

Comment:
Is there a link to where you found this code originally?

Reply:
Good call.